api/config/project/project.yaml

```diff
@@ -807,7 +819,7 @@ plugins:
             - widgets
             - '1'
       nextApiBaseUrl: $NEXT_API_BASE_URL
-      nextSecretToken: $NEXT_REVALIDATE_SECRET_TOKEN
+      nextSecretToken: $NEXT_SECRET_TOKEN
```

Comment:
Renamed `NEXT_REVALIDATE_SECRET_TOKEN` to `NEXT_SECRET_TOKEN` because it is not used in both the revalidation and reviewing context